### PR TITLE
5.03.00044 Cross points

### PR DESCRIPTION
**Pull request by Nila24**
Cross point fix

### CODE REVIEW

File: course_management.lua [hunk 1]
```diff
@@ -170,59 +174,69 @@ function courseplay:loadCourse(vehicle, id, useRealId, addCourseAtEnd) -- fn is
...
+							--Calculate actual turn direction between pair of crosspoints
+							local angleTurn = math.atan2(wp2.cx-wp1.cx,wp2.cz-wp1.cz) -- in radians 
+							--add direction change differences between original direction and turn direction and destination direction							
+							local totalAngle=math.deg(
+								   math.abs(getDeltaAngle(math.rad(wp1.angle),angleTurn)) + 
+								   math.abs(getDeltaAngle(angleTurn,math.rad(wp2.angle))))
+							angleTurn = math.deg(angleTurn) -- now in degrees
+							--courseplay:debug(string.format('course1 wp %d, course2 wp %d, dist=%s', wpNum1, wpNum2, tostring(dist)), 8);
+							if dist and dist ~= 0 and dist < wpDistMax then
+								courseplay:debug(string.format('wp1 %d %.2f° wp2 %d %.2f° dist=%.1f angleTurn %.2f°, totalAngle %.2f°, lowA %.2f°, lowD %.1f',
```
Comment (pops64) [inline course_management.lua]: Update this to `courseplay.debugVehicle(8, vehicle,'wp1 %d %.2f° wp2 %d %.2f° dist=%.1f angleTurn %.2f°, totalAngle %.2f°, lowA %.2f°, lowD %.1f', wpNum1, wp1.angle, wpNum2, wp2.angle, dist, angleTurn , totalAngle, smallestAngle, smallestDist)` thanks were trying to update the debug to that function

**Comment by Nila24** [comment]
Done

File: modDesc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" standalone="no" ?>
 <modDesc descVersion="39">
-	<version>5.03.00043</version>
+	<version>5.03.00044</version>
```
Comment (pops64) [inline modDesc.xml]: Sorry I made some pushes version 46 would be next at the time of this writing. Just waiting on a second opinion about this. Thanks for your patience

**Comment by Nila24** [comment]
Hi, always forgetting something. 
Version control is a nightmare. My version has much more changes, so I keep needing to update the one in the branch...

**Pull request merged**